ShowSQLTables/DataTaker.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Data.SqlClient;
+
+namespace ShowSQLTables
+{
+    class DataTable
+    {
+        private SqlCommand command;
+        private SqlDataReader reader;
+
+        public DataTable(SqlCommand command, SqlDataReader reader)
```

Comment:
This class will be more safe and reusable if its getting query string and open conncetion/reader on its own. That way chances of failure will be much more lower. Add IDataStore interface for this class which will support Get operation